test/jit/test_save_load_for_op_version.py

```diff
@@ -240,10 +240,6 @@ def forward(self, a, b: int):
         except Exception as e:
             self.skipTest("Failed to load fixture!")
 
-        for m in (v3_module_float, v3_module_int):
```

Comment:
Why remove this?

Reply:
This was a leftover buggy code. We moved the mobile version of upgrader tests from test_save_load. This was part of old tests. You can find them at test_upgraders_legacy.py